packages/c/citra/package.yml

```diff
@@ -11,23 +11,28 @@ description: |
     Citra is an emulator for the Nintendo 3DS handheld. It is capable of running and debugging homebrew applications, as well as many commercial games.
 builddeps  :
```

Comment:
Just one minor issue. Please fix the build order. You can see the correct order in the checks output.

Reply:
Should be good to go now